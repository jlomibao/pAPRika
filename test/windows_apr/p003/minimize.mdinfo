

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1478E+02     6.1685E-01     3.0870E+00     H3        114

 BOND    =        1.2475  ANGLE   =      108.4634  DIHED      =       54.1987
 VDWAALS =      -58.2462  EEL     =     1323.9250  EGB        =     -125.6569
 1-4 VDW =        5.0286  1-4 EEL =    -2124.4902  RESTRAINT  =        0.7516
 EAMBER  =     -815.5301
 NMR restraints: Bond =    0.686   Angle =     0.033   Torsion =     0.005
===============================================================================
