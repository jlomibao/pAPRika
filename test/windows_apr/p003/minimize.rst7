defa
  125
  -0.5208741   4.9312949   2.0241998   1.0171339   4.8744127   2.0346500
  -0.8701683   4.2247083   0.8004029  -0.8939953   4.2594675   3.2637461
  -0.9012341   5.9591140   2.0055040   1.4726781   5.8713056   2.0237161
   1.3300040   4.1457679   0.8146421   1.3236353   4.1744666   3.2768777
   2.6913555   4.0876854   0.3100181   2.7003341   4.0598040   3.7645193
   0.1993793   3.9282164   4.0261502   0.2209177   3.8057913   0.0885513
   0.1792295   3.5586554   5.1871769   0.2100279   3.2973364  -1.0194125
   2.6638254   4.0637491  -0.7854746   3.2347713   4.9752572   0.6540413
   2.6779655   4.0033002   4.8589333   3.2514843   4.9544889   3.4527362
  -4.6768315   1.9861933   1.9308422  -3.8137278   3.2598285   1.9620273
  -4.2393736   1.3176649   0.7138277  -4.3238554   1.3100698   3.1720485
  -5.7498899   2.2067200   1.8941116  -4.4171495   4.1747680   1.9477556
  -3.0122444   3.1454539   0.7534779  -3.0743560   3.1407340   3.2155197
  -2.2241240   4.2678747   0.2741130  -2.2801147   4.2560812   3.7373093
  -3.4581457   2.0416039   3.9448993  -3.2958882   2.0255447   0.0192087
  -3.1495993   1.7989899   5.0985055  -2.8463618   1.7488078  -1.0796074
  -2.1819080   4.2304649  -0.8205347  -2.7008324   5.1992854   0.6007007
  -2.2733830   4.1956199   4.8317198  -2.7537340   5.1932074   3.4228008
  -4.1071787  -3.0822350   1.8884806  -4.8507514  -1.7342307   1.8866851
  -3.2986997  -3.0153138   0.6798197  -3.3591677  -3.0554840   3.1409471
  -4.7908803  -3.9384045   1.8574730  -5.9394600  -1.8549059   1.8468536
  -4.3534447  -1.0820756   0.6838907  -4.4378513  -1.1200613   3.1418130
  -4.9458547   0.1548447   0.1995436  -5.0468418   0.1205103   3.6186238
  -3.6431966  -1.9446647   3.8957607  -3.4790454  -1.8578299  -0.0284059
  -3.3206033  -1.7595520   5.0551517  -3.0062984  -1.5975350  -1.1214706
  -4.8972631   0.1661607  -0.8954204  -5.9895945   0.2002167   0.5311833
  -5.0565804   0.1075497   4.7146039  -6.0732170   0.1733102   3.2375693
   0.5836379  -5.0782128   1.9028449  -0.9542639  -5.0181499   1.8977377
   0.9478620  -4.3580766   0.6915778   0.9452534  -4.4172008   3.1508783
   0.9631406  -6.1062182   1.8779825  -1.4127318  -6.0134228   1.8717660
  -1.2529853  -4.2740543   0.6832253  -1.2717603  -4.3285551   3.1422215
  -2.6118887  -4.1921346   0.1730800  -2.6499143  -4.2396725   3.6280299
  -0.1544604  -4.0783658   3.8992775  -0.1350067  -3.9276643  -0.0267911
  -0.1451434  -3.7088647   5.0592590  -0.1111412  -3.4025545  -1.1267667
  -2.5778369  -4.1394490  -0.9212234  -3.1622144  -5.0856058   0.4897616
  -2.6320529  -4.2088986   4.7235522  -3.1932632  -5.1303622   3.2920538
   4.7409477  -2.1330500   1.9298127   3.8819063  -3.4103790   1.9217453
   4.3115107  -1.4358137   0.7264672   4.3746426  -1.4883542   3.1840329
   5.8150971  -2.3488243   1.8969484   4.4879908  -4.3231585   1.8912368
   3.0897399  -3.2686084   0.7087461   3.1313941  -3.3223674   3.1698333
   2.3122637  -4.3823543   0.1899291   2.3213883  -4.4431745   3.6494761
   3.5064796  -2.2427674   3.9302820   3.3765115  -2.1292197   0.0059091
   3.1933214  -2.0346249   5.0883805   2.9360172  -1.8261835  -1.0896409
   2.2894211  -4.3200002  -0.9041574   2.7863553  -5.3197107   0.5030371
   2.2959796  -4.4110525   4.7448106   2.7918165  -5.3760180   3.3179092
   4.1710648   2.9365414   2.0105722   4.9097239   1.5861113   1.9827920
   3.3797150   2.9017592   0.7896520   3.4073465   2.8822337   3.2535515
   4.8583947   3.7903621   2.0087361   5.9992173   1.7037842   1.9584442
   4.4247224   0.9648084   0.7583465   4.4801890   0.9426794   3.2173896
   5.0208601  -0.2597441   0.2473730   5.0874749  -0.3079530   3.6695480
   3.6753200   1.7489031   3.9821103   3.5614151   1.7608500   0.0556477
   3.3290350   1.5307514   5.1298303   3.0989299   1.5285822  -1.0480921
   4.9802790  -0.2435105  -0.8478552   6.0622754  -0.3132084   0.5850594
   5.0841702  -0.3225236   4.7655461   6.1185108  -0.3478130   3.2996662
   0.0130005  -0.0349112   2.8063292  -0.9060844  -0.6092701   2.6372602
   0.8705761  -0.6963130   2.6322349   0.0504219   0.7744853   2.0669598
   0.0452397   0.5407880   4.2292323   0.9586044   1.1357546   4.3515815
  -0.0059307  -0.5630106   5.3035209  -0.8048627   1.2221081   4.3569788
  -0.9168906  -1.1595736   5.1714064   0.8430953  -1.2441319   5.1678047
   0.0236729   0.0037658   6.7356671  -0.8346843   0.6635110   6.9121821
  -0.0138528  -0.8090123   7.4713155   0.9416114   0.5786568   6.9090999
   0.0010116  -0.0012378  -6.0229530  -0.0023966   0.0032386  -9.0000586
  -0.0009375   2.2000051 -11.2000212
