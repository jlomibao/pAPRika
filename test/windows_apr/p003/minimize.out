
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:17:36

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/p003
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   410822 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.8990E+02     1.0438E+01     2.6756E+01     H4        116

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -45.6804  EEL     =     1326.5537  EGB        =     -122.4732
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -789.8978
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1379E+02     6.3635E-01     3.5019E+00     H3        114

 BOND    =        1.2336  ANGLE   =      108.5454  DIHED      =       54.3579
 VDWAALS =      -56.4901  EEL     =     1323.0460  EGB        =     -125.4141
 1-4 VDW =        4.9428  1-4 EEL =    -2124.0553  RESTRAINT  =        0.0441
 EAMBER  =     -813.8338
 NMR restraints: Bond =    0.023   Angle =     0.018   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1391E+02     6.2783E-01     3.4051E+00     H3        114

 BOND    =        1.2540  ANGLE   =      108.5525  DIHED      =       54.3329
 VDWAALS =      -56.6953  EEL     =     1323.3583  EGB        =     -125.4894
 1-4 VDW =        4.9642  1-4 EEL =    -2124.2660  RESTRAINT  =        0.0759
 EAMBER  =     -813.9888
 NMR restraints: Bond =    0.047   Angle =     0.024   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1401E+02     6.2530E-01     3.3566E+00     H3        114

 BOND    =        1.2531  ANGLE   =      108.5493  DIHED      =       54.3334
 VDWAALS =      -56.8569  EEL     =     1323.3882  EGB        =     -125.4888
 1-4 VDW =        4.9718  1-4 EEL =    -2124.2780  RESTRAINT  =        0.1144
 EAMBER  =     -814.1281
 NMR restraints: Bond =    0.082   Angle =     0.026   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1411E+02     6.2594E-01     3.3223E+00     H3        114

 BOND    =        1.2487  ANGLE   =      108.5438  DIHED      =       54.3345
 VDWAALS =      -57.0175  EEL     =     1323.3965  EGB        =     -125.4818
 1-4 VDW =        4.9789  1-4 EEL =    -2124.2748  RESTRAINT  =        0.1627
 EAMBER  =     -814.2718
 NMR restraints: Bond =    0.127   Angle =     0.027   Torsion =     0.002
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1421E+02     6.3856E-01     3.2895E+00     H3        114

 BOND    =        1.2359  ANGLE   =      108.5355  DIHED      =       54.3309
 VDWAALS =      -57.1968  EEL     =     1323.3980  EGB        =     -125.4606
 1-4 VDW =        4.9883  1-4 EEL =    -2124.2698  RESTRAINT  =        0.2270
 EAMBER  =     -814.4387
 NMR restraints: Bond =    0.188   Angle =     0.028   Torsion =     0.002
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1432E+02     6.2238E-01     3.2569E+00     H3        114

 BOND    =        1.2550  ANGLE   =      108.5257  DIHED      =       54.3208
 VDWAALS =      -57.3808  EEL     =     1323.4840  EGB        =     -125.5216
 1-4 VDW =        4.9912  1-4 EEL =    -2124.2927  RESTRAINT  =        0.3026
 EAMBER  =     -814.6185
 NMR restraints: Bond =    0.259   Angle =     0.028   Torsion =     0.003
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1443E+02     6.2600E-01     3.2195E+00     H3        114

 BOND    =        1.2686  ANGLE   =      108.5131  DIHED      =       54.3019
 VDWAALS =      -57.5862  EEL     =     1323.5850  EGB        =     -125.5775
 1-4 VDW =        4.9965  1-4 EEL =    -2124.3267  RESTRAINT  =        0.3966
 EAMBER  =     -814.8252
 NMR restraints: Bond =    0.349   Angle =     0.029   Torsion =     0.003
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1454E+02     6.3105E-01     3.1804E+00     H3        114

 BOND    =        1.2750  ANGLE   =      108.4990  DIHED      =       54.2762
 VDWAALS =      -57.7912  EEL     =     1323.6934  EGB        =     -125.6235
 1-4 VDW =        5.0036  1-4 EEL =    -2124.3703  RESTRAINT  =        0.4990
 EAMBER  =     -815.0378
 NMR restraints: Bond =    0.446   Angle =     0.030   Torsion =     0.004
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1466E+02     6.1827E-01     3.1364E+00     H3        114

 BOND    =        1.2588  ANGLE   =      108.4822  DIHED      =       54.2417
 VDWAALS =      -58.0119  EEL     =     1323.7896  EGB        =     -125.6308
 1-4 VDW =        5.0161  1-4 EEL =    -2124.4199  RESTRAINT  =        0.6177
 EAMBER  =     -815.2742
 NMR restraints: Bond =    0.559   Angle =     0.031   Torsion =     0.004
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1478E+02     6.1685E-01     3.0870E+00     H3        114

 BOND    =        1.2475  ANGLE   =      108.4634  DIHED      =       54.1987
 VDWAALS =      -58.2462  EEL     =     1323.9250  EGB        =     -125.6569
 1-4 VDW =        5.0286  1-4 EEL =    -2124.4902  RESTRAINT  =        0.7516
 EAMBER  =     -815.5301
 NMR restraints: Bond =    0.686   Angle =     0.033   Torsion =     0.005
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1478E+02     6.1685E-01     3.0870E+00     H3        114

 BOND    =        1.2475  ANGLE   =      108.4634  DIHED      =       54.1987
 VDWAALS =      -58.2462  EEL     =     1323.9250  EGB        =     -125.6569
 1-4 VDW =        5.0286  1-4 EEL =    -2124.4902  RESTRAINT  =        0.7516
 EAMBER  =     -815.5301
 NMR restraints: Bond =    0.686   Angle =     0.033   Torsion =     0.005
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.431    9.302    0.130    0.084 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.744  147.398    0.347    0.004 a
  DUM  DM2    4 --  DUM  DM1    3:   49.542   49.483    0.059    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.586   62.003    0.584    0.010 a
  DUM  DM1    3 --  C7   CB6    1: -146.072 -146.178    0.106    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.179  -72.548    0.369    0.004 t
  DUM  DM1    3 --  C    BUT    2:    8.829    9.176    0.347    0.602 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.867  180.000    0.133    0.001 a
  DUM  DM1    3 --  C3   BUT    2:  179.214  180.000    0.786    0.019 a
                                       Total distance penalty:      0.686
                                       Total angle    penalty:      0.033
                                       Total torsion  penalty:      0.005
|                               RMS deviation from ideal bonds :      0.0096
|                               RMS deviation from ideal angles:      4.483
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.32 (22.23% of Gen B)
|                Calc gb diag               4.10 (39.23% of Gen B)
|                Calc gb off-diag           4.02 (38.52% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.44 (100.0% of Nonbo)
|          Nonbond force             10.44 (90.45% of Force)
|          Bond/Angle/Dihedral        1.07 ( 9.26% of Force)
|          Other                      0.03 ( 0.29% of Force)
|       Force time                11.54 (100.0% of Runmd)
|    Runmd Time                11.54 (99.73% of Total)
|    Other                      0.03 ( 0.26% of Total)
| Total time                11.57 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:17:36.407  on 04/10/2019
|           Setup done at 18:17:36.412  on 04/10/2019
|           Run   done at 18:17:47.980  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
